theme/static/js/generic-resource.js

```diff
@@ -296,6 +296,26 @@ $(document).ready(function () {
         })
     });
 
+    $("#new-version-btn").on('click', function(e) {
```

Comment:
It looks like before you added this, someone could hit the new version button multiple times?  This looks good, just would like to confirm why you made this change.

Reply:
Yes, I added code to disable new version button once it is clicked, so it cannot be clicked multiple times.